pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -80,27 +80,29 @@ func (sv *openShiftClusterStaticValidator) validateProperties(path string, p *Op
 	default:
 		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, path+".provisioningState", "The provided provisioning state '%s' is invalid.", p.ProvisioningState)
 	}
+	if isCreate {
```

Comment:
currently the function flow very closely matches the field order in the struct; I think it is worth keeping that for maintainability.  I suggest multiple `if isCreate {}`s.